typos.toml

```diff
@@ -34,6 +34,7 @@ targetting = "targetting"
 publically = "publically"
 clonable = "clonable"
 moreso = "moreso"
+EnzymeTypeTreeShiftIndiciesEq = "EnzymeTypeTreeShiftIndiciesEq"
```

Comment:
https://rust-lang.zulipchat.com/#narrow/channel/182449-t-compiler.2Fhelp/topic/.E2.9C.94.20tidy.20error.3A.20typo.20EnzymeTypeTreeShiftIndiciesEq.20to.20Enzym.2E.2E.2E/near/536350657: default.extend-identifiers, not default.extend-words